lib/avrora/http_client.ex

```diff
@@ -42,4 +42,6 @@ defmodule Avrora.HTTPClient do
       {:error, _} -> {:error, {status, body}}
     end
   end
+
+  defp ssl_options, do: [verify: :verify_peer, cacerts: :public_key.cacerts_get()]
```

Comment:
@goozzik Can you please take a look on those errors below?

I'm going to push an additional test matrix combinations to cover OTP 25

Reply:
@Strech `:public_key.cacerts_get()` was added in OTP 25. I added condition that will execute at compile time and will fallback to previous version (no ssl options) for OTP versions below 25.

https://github.com/erlang/otp/commit/8f0270a35d56954987331d7c2833802cf3a7058a

Reply:
I've checked the [OTP source code](https://github.com/erlang/otp/blob/master/lib/public_key/src/pubkey_os_cacerts.erl#L133-L163) together with [some issues in their repo](https://github.com/erlang/otp/issues/5899) and I think we can't go with this fix because it introduces backward incompatible change.

By default (before OTP 25) it was `{verify: :verify_none}` and you are changing it to `{verify: :verify_peer}` now, as a quick-fix I would explicitly set it to `:verify_none` now (it will be OTP version independent) and later we can come up with a setting to enable SSL verification

Reply:
I've created a follow-up issue to keep track https://github.com/Strech/avrora/issues/98

Reply:
You are right, I changed it to `verify_none` https://github.com/Strech/avrora/pull/97/commits/b83b92b466beba75469007af7696ab24a86e3174